src/stories/ui/tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { Tooltip } from '../../../sections/ui/tooltip/Tooltip'
+
```

Comment:
It would be nice to add some docs. I wasn't sure how to do this myself but I think I figured it out:

```
 import { Tooltip } from '../../../sections/ui/tooltip/Tooltip'
 
+/**
+ * ## Description
+ * Blah blah blah...
+ *
+*/
+
 const meta: Meta<typeof Tooltip> = {
   title: 'UI/Tooltip',
-  component: Tooltip
+  component: Tooltip,
+  tags: ['autodocs']
 }
```

## before

![Screen Shot 2023-04-18 at 4 30 11 PM](https://user-images.githubusercontent.com/21006/232898070-96933174-4818-4529-91f8-6ccc1fd50c0b.png)

## after

![Screen Shot 2023-04-18 at 4 30 31 PM](https://user-images.githubusercontent.com/21006/232898122-f152a805-b2d1-4e31-9689-d521a58e7fb7.png)

Reply:
ok, I added documentation.